.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["eslint:recommended", "fliplet"],
+  "globals": {
+    "CodeMirror": "readonly",
```

Comment:
These are global values used by https://github.com/Fliplet/fliplet-widget-data-source/blob/master/.eslintrc but do not apply to this repo